lease/lessor.go

```diff
@@ -46,7 +46,8 @@ type DeleteableRange interface {
 // a lessor is the owner of leases. It can grant, revoke,
 // renew and modify leases for lessee.
 // TODO: use clockwork for testability.
-type lessor struct {
+// TODO: create an interface for Lessor?
+type Lessor struct {
```

Comment:
I will change it to Lessor interface when I start to implement the keepAlive and expiration functionality. Or you want me to change this to an interface now?